tests/admin_filters/tests.py

```diff
@@ -554,23 +559,55 @@ def test_relatedfieldlistfilter_foreignkey(self):
         self.assertIs(choice['selected'], True)
         self.assertEqual(choice['query_string'], '?author__id__exact=%d' % self.alfred.pk)
 
-    def test_relatedfieldlistfilter_foreignkey_ordering(self):
+    def test_relatedfieldlistfilter_foreignkey_admin_ordering(self):
         """RelatedFieldListFilter ordering respects ModelAdmin.ordering."""
-        class EmployeeAdminWithOrdering(ModelAdmin):
-            ordering = ('name',)
+        class OfficeAdminWithOrdering(ModelAdmin):
+            ordering = ('code',)
 
-        class BookAdmin(ModelAdmin):
-            list_filter = ('employee',)
+        class EmployeeAdmin(ModelAdmin):
+            list_filter = ('office',)
 
-        site.register(Employee, EmployeeAdminWithOrdering)
-        self.addCleanup(lambda: site.unregister(Employee))
-        modeladmin = BookAdmin(Book, site)
+        site.register(Office, OfficeAdminWithOrdering)
+        self.addCleanup(lambda: site.unregister(Office))
+        modeladmin = EmployeeAdmin(Employee, site)
 
         request = self.request_factory.get('/')
         request.user = self.alfred
         changelist = modeladmin.get_changelist_instance(request)
         filterspec = changelist.get_filters(request)[0][0]
-        expected = [(self.jack.pk, 'Jack Red'), (self.john.pk, 'John Blue')]
+        # Expected to be sorted by `code`.
+        expected = [
+            (self.black_office.pk, str(self.black_office)),
+            (self.blue_office.pk, str(self.blue_office)),
+            (self.red_office.pk, str(self.red_office)),
+        ]
+        self.assertEqual(filterspec.lookup_choices, expected)
+
+    def test_relatedfieldlistfilter_foreignkey_model_ordering(self):
+        """
+        RelatedFieldListFilter ordering respects Model.ordering when
+        no ordering is defined in ModelAdmin.
+        """
+        class OfficeAdminWithOrdering(ModelAdmin):
+            pass
+
```

Comment:
We can register model without defining `OfficeAdminWithOrdering` with the same effect:
```python
site.register(Office)
```